src/components/radio-button/__spec__.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import TestUtils from 'react-dom/test-utils';
+import RadioButton from './radio-button';
+import { shallow } from 'enzyme';
+import { elementsTagTest, rootTagTest } from '../../utils/helpers/tags/tags-specs';
+
+describe('RadioButton', () => {
+  let instance;
+
+  beforeEach(() => {
+    instance = TestUtils.renderIntoDocument(
+      <RadioButton
+        name='radiobutton'
+        label='radiobutton'
+      />
+    )
+  });
+
+  describe('render', () => {
+    it('renders a parent div with a pod CSS class', () => {
+      let radioButtonNode = TestUtils.scryRenderedDOMComponentsWithTag(instance, 'div')[0];
```

Comment:
(I realise this PR is just restoring the deleted files, so feel free to ignore this comment.)

I think every `let` in this file can be changed to `const` (except for `let instance` on line 8), because their values are never changed.

This should be picked up by https://eslint.org/docs/rules/prefer-const